### PR TITLE
chore: clean up really old python code

### PR DESCRIPTION
**Pull request by KevinBoyette**
- generators are a thing to be used
- collections has had the Counter class forever.

### CODE REVIEW

File: algorithms/sock_merchant/kevin/kevin.py [hunk 1]
```diff
@@ -1,17 +1,28 @@
-#!/bin/python3
+#!usr/bin/env python3
 # Kevin Boyette
 import sys
```
Comment (prithajnath) [inline algorithms/sock_merchant/kevin/kevin.py:3]: Probably don't need this anymore either

**Review by prithajnath — APPROVED** [review]

File: algorithms/sock_merchant/kevin/kevin.py (hunk 1 above)
Comment (AlexLanzano) [reply algorithms/sock_merchant/kevin/kevin.py:3]: He does if he wants to run it from command line without prefixing with python

Comment (prithajnath) [reply algorithms/sock_merchant/kevin/kevin.py:3]: @AlexLanzano I meant the `import sys` line not the shebang

Comment (KevinBoyette) [reply algorithms/sock_merchant/kevin/kevin.py]: @prithajnath @AlexLanzano I cleaned it up a bit more.  Thanks for catching the leftover import!

Comment (AlexLanzano) [reply algorithms/sock_merchant/kevin/kevin.py]: Oh, Im stupid. Carry on

**Pull request merged**